straxen/get_corrections.py

```diff
@@ -91,6 +93,11 @@ def get_correction_from_cmt(run_id, conf):
             else:
                 return float(correction) # float elife, drift velocity, etc
         
+        elif model_conf in arrays_corrections:
+            np_correction = correction.reshape(correction.size)
+            np_correction = np_correction.astype(np.int16)  # not sure if straxen can handle dtype:object therefore specify dtype
+            return np_correction
+        
```

Comment:
Hi Daniel

The thing that makes things tricky is the fact that any Numpy types are forging to mongoDB, therefore we cannot save the hit_thresholds as np.array, one potential solution is to have one dataframe for every channel but this is not convenient since there are more that 2000s channels in some cases. To workaround this the hit thresholds are saved as list() in the DB, but then straxen is expecting a Numpy type. 
In `hitfinder_thresholds.py` also all the thresholds are defined as dtype=np.int16. 
The other solution I can see is, do the cast form list to np.array at the level of plugin... which don't think is the best solution but can't think of another solution, any other suggestion?

For the future we need to think how we handled the fact that Numpy types are forging to mongoDB, since as you mentioned that we may want to save corrections arrays that are not of dytype=np.int16